refinery/data_set_manager/tasks.py

```diff
@@ -298,13 +306,15 @@ def generate_auxiliary_file(auxiliary_node, parent_node_file_store_item):
         # Here we are checking for the FileExtension of the ParentNode's
         # FileStoreItem because we will create auxiliary files based on what
         # said value is
-        if parent_node_file_store_item.get_extension().lower() == 'bam':
+        if parent_node.file_item.get_extension().lower() == 'bam':
```

Comment:
Is this redundant given the check here: https://github.com/refinery-platform/refinery-platform/pull/3484/files#diff-10b1a3c979459ad2810c191d2321daa3R1427

Reply:
i suppose currently it is unnecessary but going forward, especially given the name of the function, different behaviors for different functions should be possible.